src/foundation.cs

```diff
@@ -14866,6 +14866,9 @@ interface NSTask {
 		[Export ("launch")]
 		void Launch ();
 
+		[Export ("launchAndReturnError:")]
+		bool LaunchAndReturnError ([NullAllowed] out NSError error);
```

Comment:
C# has method overloading, so this could be just:

```suggestion
		bool Launch ([NullAllowed] out NSError error);
```